src/utils/json-data.js

```diff
@@ -0,0 +1,135 @@
+const fs = require('fs-extra');
+const path = require('path');
+const reserved = require('reserved-words');
+const shared = require('./shared');
+const glob = require('glob');
+const BUILD_TIME_BINDING = '@buildtime:';
+// Matches {{ stache.jsonData. }} with any number of spaces between the '{{' and 'stache.',
+// any keys following the 'jsonData.', and the closing '}}'
+const angularBindingRegExp = new RegExp(/\{\{\s*stache\.jsonData\..*?\}\}/g);
+const buildTimeBindingRegExp = new RegExp(`\\{\\{\\s*${BUILD_TIME_BINDING}\\s*stache\\.jsonData\\..*?\\}\\}`, 'g');
```

Comment:
It does because it's a string being passed in.   It needs to escape the escape character, so when it renders as a regex,  it comes out as /\{\{\s*@buildtime: etc...  The string is needed so we can use that variable inside it.